components/src/core/channel-value/channel-value.tsx

```diff
@@ -104,25 +117,47 @@ export const ChannelValue: React.FC<ChannelValueProps> = (props) => {
         }
     }, [icon, fontSize, getColor, iconColor, iconSize]);
 
-    const getUnits = useCallback((): JSX.Element | undefined => {
-        if (units) {
-            return (
-                <Body1 font={'light'} fontSize={fontSize} style={[{ color: getColor() }, styles.units]}>
-                    {units}
-                </Body1>
-            );
-        }
-    }, [units, fontSize, getColor, styles]);
+    const getUnits = useCallback(
+        (spacerLocation: 'before' | 'after'): JSX.Element | undefined => {
+            if (units) {
+                return (
+                    <>
+                        {((spacerLocation === 'before' && unitSpace === 'show') ||
+                            (spacerLocation === 'before' &&
+                                unitSpace === 'auto' &&
+                                !suffixUnitWhitelist.includes(units))) && <Spacer flex={0} width={4} />}
```

Comment:
this is designed to be `0.25em` instead of 4px (there is a screenshot somewhere in the blue skies chat about it). Can you do the calculation using `fontSize`?